s08compute_wct.py

```diff
@@ -0,0 +1,425 @@
+"""
+WCT
```

Comment:
would ideally need some more documentation, but I reckon it could be in another PR (I'm working on a automated documentation builder that would allow checking the PRs)